cmd/api-response.go

```diff
@@ -267,8 +267,7 @@ func (o ObjectVersion) MarshalXML(e *xxml.Encoder, start xxml.StartElement) erro
 	} else {
 		start.Name.Local = "Version"
 	}
-	type objectVersionWrapper ObjectVersion
-	return e.EncodeElement(objectVersionWrapper(o), start)
+	return e.EncodeElement(o, start)
```

Comment:
The wrapper is to avoid infinite recursion, otherwise we are just calling ourselves.